scripts/build_snp.sh

```
@@ -0,0 +1,183 @@
+    fi
+    popd
+}
+
+build_qemu() {
+    local build_dir=$1
+    pushd ${build_dir}
+    git clone --single-branch -b ${QEMU_BRANCH} ${QEMU_GIT_URL} qemu
+    cd qemu
+
+    ./configure --enable-slirp --enable-kvm --target-list=x86_64-softmmu
+    make -j$(nproc)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)